app/src/components/Forms/Info/DetailedInfo.js

```diff
@@ -0,0 +1,247 @@
+
+          <h3>Coming soon</h3>
+          <ul>
+            <li>Information on non-participating candidates</li>
+            <li>
+              Further integration with the{' '}
+              <a
+                href="civicplatform.org"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                CIVIC Platform
+              </a>{' '}
+              to facilitate public use and sharing of data visualizations
+            </li>
+            <li>
+              Documentation of the collaborative software development process
+            </li>
+          </ul>
+        </Collapsable.Section>
+      </Collapsable>
+    </section>
+    <section css={contentStyle}>
+      <h2>About the data</h2>
+      <p>
+        This data is pulled live from the Open and Accountable Elections
+        application. The dashboard shows all contribution information that has
+        been submitted by campaigns, including contributions that are awaiting
+        City review.
+      </p>
+      <Collapsable description="About this data">
+        <Collapsable.Section hidden>
+          <h3>Contribution process</h3>
+          <p>
+            A campaign submits contribution information through the Open and
+            Accountable Elections application. City staff then review the
+            contribution for matching eligibility and proper documentation, and
+            may request clarification or changes from the campaign. Once any
+            issues have been resolved, an eligible contribution can be approved
+            for matching by City staff. An additional layer of review happens
+            prior to payment from the Open & Accountable Elections Fund.
+          </p>
+          <p>
+            Contributions are shown on the dashboard as soon as they are
+            submitted by a campaign. Contribution information may be revised
+            during the review process. The match information shown is what has
+            been approved for matching by City staff, not what has actually been
+            paid.
+          </p>
+          <h3>Public information and privacy</h3>
+          <p>
+            Although names and addresses of individual contributors is public
+            information, the purpose of this dashboard is to show aggregate
+            information. The map limits your ability to zoom in on precise
+            locations and does not display contributor names.
+          </p>
+          <h3>Additional notes</h3>
+          <p>
+            The information above includes both monetary and non-monetary
+            (in-kind) contributions. Program rules limit in-kind contributions
+            to $20,000 per campaign.
+          </p>
+          <p>
+            When a contribution is on the border of a size category, it is
+            grouped into the smaller category. So a $100 contribution would be
+            included in the $25-$100 category, not the $100-$250 category.
```

Comment:
Why not just rename these sections to be accurate? i.e., "$101-$250"

Reply:
To be fully accurate, we'd need to show the amount to the penny. I don't love the way it is, but keeping it for now